models/GET-models.js

```diff
@@ -52,7 +52,31 @@ const fetchArticleId = (id) => {
     })
 }
 
+const fetchCommentsByArticleId = (id) => {
+
+    if (Number.isNaN(id)) {
+        return Promise.reject({ status: 400, msg: "Bad Request" });
+    }
+
+    let str = `SELECT * FROM comments
+               WHERE comments.article_id = $1;`
+
+    return db.query(str, [id]).then(({ rows }) => {
+
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "Resource not found" });
+        }
+
+        rows.map((comment) => {
```

Comment:
Instead of SELECT * and then mapping over the results to remove a property - I'd recommend tweaking your SQL query to omit the `article_id` properties